packages/marketplace/src/components/ReadmePage.tsx

```diff
@@ -107,7 +107,6 @@ class ReadmePage extends React.Component<
   }
 
   render() {
-    this.props.data.isInstalled();
```

Comment:
Didn't you add this with the install button thing? I was assuming you needed the code to run for it to render or something, but I guess it's not necessary?

Reply:
It was when I was testing things, the function returns a boolean value that was assigned as state later. Having this actually breaks Marketplace occasionally.